services/portal/studyViewer/studyViewerProps.js

```diff
@@ -1,17 +1,26 @@
 module.exports = {
   path: '/study-viewer/clinical_trials',
   datasetPath: '/study-viewer/clinical_trials/ACTT',
+  datasetPath1: 'study-viewer/clinical_trials/NCT04401579',
   studyViewerDivClass: '.study-viewer',
+  // TODO : improve the selector
+  studyViewerRelPath: '//body/div[@id=\'root\']/div[1]/div[1]/div[3]/div[1]/div[1]',
   datasetDivClass: '.study-viewer__title',
-  detailedButtonXPath: '.ant-collapse-header',
-  modalDivClass: '.ant-modal-header',
+  detailedButtonXPath: '.ant-collapse-header:nth-child(1)',
+  detailedButtonXPath1: '//body/div[@id=\'root\']/div[1]/div[1]/div[3]/div[1]/div[1]/div[2]/div[2]/div[1]/'
```

Comment:
`datasetPath` / `datasetPath1` and `detailedButtonXPath` / `detailedButtonXPath1` => these could use more explicit names so the code is easier to read, my suggestion would be "dataset1Path", "dataset2Path", "dataset1DetailedButtonXPath", "dataset2DetailedButtonXPath"